Sources/Native/NativeMockServerWrapper.swift

```diff
@@ -2,102 +2,206 @@ import Foundation
 import SwiftyJSON
 import BrightFutures
 
-class NativeMockServerWrapper: MockServer {
-  var port: Int32 = -1
-  var pactDir: String = ""
-
-  public init() {
-    pactDir = defaultPactDir()
-    port = randomPort()
-  }
-
-  func defaultPactDir() -> String {
-    if let dir = ProcessInfo.processInfo.environment["pact_dir"] {
-      return dir
+#if SWIFT_PACKAGE
+    import SwiftPactServer
+#endif
+
+public class NativeMockServerWrapper: MockServer {
```

Comment:
It is really hard to read through this class where public and private declarations are all over the place. It is hard to read and learn about interface the caller needs to interact with.

Reply:
I think it's mostly private except the `MockServer` protocol implementations.
What would you do to improve it?